server/models/Posts.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const postSchema = new Schema({
```

Comment:
Can remove jobsNeeded array . Post is the model for a single Job Post. Under Customer you define a field 'posts' which is the array of Post models linked to that Customer.